src/main/java/Duke.java

```diff
@@ -1,10 +1,373 @@
+        Task task = new Task(command);
+        task.isCompleted = false;
+        task.taskName = command;
+        System.out.println("added:  " + task.taskName);
+        tasks[numberOfTasks] = task;
+    }
+
+    private static void createEvent(String command, int numberOfTasks) {
+        String content = command.substring(5);
+        String[] contents = content.split("/");
+        String title = contents[0];
+        String startTime = contents[1];
+        String endTime = contents[2];
+        Event newEvent = new Event(title);
+        newEvent.setStartTime(startTime);
+        newEvent.setEndTime(endTime);
+        tasks[numberOfTasks] = newEvent;
+        newEvent.newEventResponse();
+    }
+
+    private static void createDeadline(String command, int numberOfTasks) {
+        String content = command.substring(8);
+        String[] contents = content.split("/");
+        String title = contents[0];
+        String dueDate = contents[1];
+        Deadline newDeadline = new Deadline(title);
+        newDeadline.setEndTime(dueDate);
+        tasks[numberOfTasks] = newDeadline;
+        newDeadline.newDeadlineResponse();
+    }
+
+    private static void markTask(String command, int numberOfTasks) {
+        String[] result = command.split(" ");
+        int taskNum = Integer.parseInt(result[1]);
+        if (taskNum > numberOfTasks || taskNum <= 0) {
+            System.out.println("Error! No such task exists!");
+        } else {
+            Task target = tasks[taskNum - 1];
+            target.markAsDone();
+            System.out.println("Nice! I've marked this task as done:");
+            System.out.println(target.getStatusIcon() + "   " + target.taskName);
+        }
+    }
+
+    private static void unmarkTask(String command, int numberOfTasks) {
+        String[] result = command.split(" ");
+        int taskNum = Integer.parseInt(result[1]);
+        if (taskNum > numberOfTasks || taskNum <= 0) {
+            System.out.println("Error! No such task exists!");
+        } else {
+            Task target = tasks[taskNum - 1];
+            target.markAsNotDone();
+            System.out.println("OK, I've marked this task as not done yet:");
+            System.out.println(target.getStatusIcon() + "   " + target.taskName);
+        }
+    }
+
+    private static void printTaskList() {
+        int i = 1;
+        for (Task a : Duke.tasks) {
+            if (a != null) {
+                System.out.print(i);
+                a.printTask();
+                i++;
+            }
+        }
+
+
+    }
+
+
+    private static void printHorizontalBar() {
+        String horizontalBar = "---------------------------------------------------\n";
+        System.out.println(horizontalBar);
+    }
+
+    private static String readTask() {
+        String line;
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
+        return line;
+
+    }
+
+    private static String checkForAdditionalTask() {
```

Comment:
Perhaps you could split the code into separate classes due to the length of the main class, for better code quality.